src/utils/index.js

```diff
@@ -193,6 +193,25 @@ export default {
     return true
   },
 
+  regexCompare (value, search) {
+    try {
+      const regexpParts = search.match(/^\/(.*?)\/([gim]*)$/)
+      let regex
+
+      if (regexpParts) {
```

Comment:
What about to use the ternary condition?

Reply:
done